group_vars/logs_library/vars.yml

```diff
@@ -2,10 +2,10 @@
 stack_prefix: 'logs'
 stack_domain: ''
 stack_name: "{{ stack_prefix }}_library"  # stack_name must match the name of the folder that contains this vars.yml file.
-cloud_image: CentOS 7
+os_distribution: 'rocky9'
+cloud_image: RockyLinux-9.3_xfs
 cloud_user: centos
```

Comment:
Recently ran into this var as well: do we actually use `cloud_user` somewhere? If we do need it than it must be updated for Rocky 9.

Reply:
I don't remember, but I think so. Is it not in the OS? When we deploy instances you need to provide the image.
Anyhow. I don't want to spend time with this. Accept it or remove, let's just focus on the PR topics.
If we remove the `cloud_image` variable, the copy find will do its job.

Reply:
We cannot remove `cloud_image`! That is required for the playbook to function. I was talking about `cloud_user`, which is not updated in the PR: `centos` will not work on Rocky 9. But as far as I can find it is not used in the code. We only need it on the commandline (ansible-playbook -u cloud_user ....) when we deploy `single_group_playbooks/init.yml` for the first time to create the admin users.

Reply:
Maybe we just put `cloud_user` in the group_vars to _document_ which account to use, but I cannot remember.